### DISCUSSION TITLE
Certificate issue

### DISCUSSION THREAD

**Discussion by rzaq-alshoshan** [community/community#55925] [Projects and Issues]
### Select Topic Area

Bug

### Select Feature Area

- [X] Issues
- [ ] Projects

### Body

Hi there,

It seems there is an issue in https://objects.githubusercontent.com/ certificate.

Each time I try to download some resources, it give me the following error.
![image](https://github.com/community/community/assets/28965045/6070b23d-1d64-4b23-b0c2-8d6c3ac68fcf)

Thanks

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by airtower-luna** [comment]
What certificate do you actually see there? I'm getting a valid one, and the root CA behind it is the same as for the certificate I see for `github.com`.

**Reply by rzaq-alshoshan** [reply]
It seems an issue in my network while contacting root certificate. Changed the network and everything worked again.

Thanks for your help.

**Comment by rzaq-alshoshan** [comment]
It seems an issue in my network while contacting root certificate. Changed the network and everything worked again.

Closing the issue.
